cypress/integration/executions_spec.js

```diff
@@ -69,5 +70,98 @@ describe('Dashboard Executions Page', () => {
         });
       });
     });
+
+    it('should show a single execution', () => {
+      cy.contains('nav li a', 'Executions').as('executions');
+      cy.get('@executions').should('have.attr', 'href', '#/executions');
+      cy.get('@executions').click();
+
+      cy.url().should('include', 'executions');
+      cy.contains('.heading--xlarge', 'Executions');
```

Comment:
These first few assertions all depend on the result of the previous test. They also seem redundant with the previous test. I would remove them and start this test with a `cy.visit` to the page for a single execution